### PR TITLE
Stabilization of long e2e tests

### PR DESCRIPTION
**Pull request by kaweue**
- increased timeout since in case of ssd drive scaling and upgrading of cluster takes more time than in case of RAM drive
- taint only worker nodes - for contrail/openstack services all nodes are equal, however k8s may not work correctly if master node has taint
- in command HA test scale only command and psql

### CODE REVIEW

**Comment by kaweue** [comment]
recheck ha

File: test/e2e/ha/command_test.go [hunk 1]
```diff
@@ -289,34 +277,48 @@ func assertCommandAndDependenciesReplicasReady(t *testing.T, w wait.Wait, r int3
...
 		t.Parallel()
-		assert.NoError(t, w.ForReadyStatefulSet("webui-webui-statefulset", r))
+		assert.NoError(t, w.ForReadyStatefulSet("webui-webui-statefulset", 1))
 	})
 }
 
 func getHACommandCluster(namespace, nodeLabel, storagePath string) *contrail.Manager {
 	trueVal := true
+	one := int32(1)
+	commanConfig := contrail.PodConfiguration{
```
Comment (katrybacka) [inline test/e2e/ha/command_test.go:309]: common*

File: test/e2e/ha/command_test.go [hunk 2]
```diff
@@ -388,7 +387,7 @@ func getHACommandCluster(namespace, nodeLabel, storagePath string) *contrail.Man
 		Spec: contrail.PostgresSpec{
 			ServiceConfiguration: contrail.PostgresConfiguration{
 				Storage: contrail.Storage{
-					Path: storagePath + uuid.New().String(),
+					Path: storagePath + "psql",
```
Comment (katrybacka) [inline test/e2e/ha/command_test.go:390]: why is this changed?

File: test/env/create_k8s_cluster.sh [hunk 3]
```diff
@@ -45,6 +45,7 @@ echo "$kindConfig" | kind create cluster --name "${kind_cluster_name}" --config=
 cmd="echo ${insecure_registry_address} registry >> /etc/hosts"
 for node in $(kind get nodes --name "${kind_cluster_name}"); do
   docker exec "${node}" sh -c "${cmd}"
+  docker exec "${node}" sh -c "systemctl restart containerd"
```
Comment (katrybacka) [inline test/env/create_k8s_cluster.sh:48]: is that really necessary, it happend only one for you that you had and random issue with containerd

**Review by Dysproz — APPROVED** [review]

File: test/e2e/ha/command_test.go (hunk 1 above)
Comment (kaweue) [reply test/e2e/ha/command_test.go]: done

**Review by katrybacka — APPROVED** [review]

**Comment by kaweue** [comment]
recheck ha
